lib/kubernetes-deploy/runner.rb

```diff
@@ -136,7 +136,9 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       success = false
     ensure
       @logger.print_summary(success)
-      ::StatsD.measure('duration.total', (Time.now.utc - start) * 1000.0, tags: statsd_tags)
+      status = success ? "success" : "failed"
+      ::StatsD.increment('status.total', 1, tags: statsd_tags << "status:#{status}")
+      ::StatsD.measure('duration.total', (Time.now.utc - start), tags: statsd_tags << "status:#{status}")
```

Comment:
When exploring this metric in practice I got momentarily confused by `total`--it sounds like it is summing something. Wdyt about changing it to `full_deploy` or `all_resources` or `overall` or something like that?